mne_bids/write.py

```diff
@@ -1887,6 +1888,8 @@ def write_anat(image, bids_path, landmarks=None, deface=False,
     return bids_path
 
 
+@deprecated(extra='mark_bad_channels is deprecated in favor of mark_channels '
+                  'and will be removed in v1.0.')
```

Comment:
you mean 0.10 ?

Reply:
And here I am, disappointed again, because for a split second I believed our 1.0 release was imminent 😅